MyStore/src/include/StorageData.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+#include "GeneratorData.h"
+#include "JsonParcer.h"
+
+class StorageData
+{
+public:
+	std::vector<InfoProduct> m_data;
+	const int m_max_size_data = 5000;
+
+	JsonParcer m_Parcer = ("..\\MyStore\\data\\Data.json");

```

Comment:
This is interesting. :) I prefer to avoid such implicit constructors, but you don't need to change this.